mllib-local/src/main/scala/org/apache/spark/ml/linalg/Vectors.scala

```diff
@@ -240,6 +240,8 @@ sealed trait Vector extends Serializable {
 @Since("2.0.0")
 object Vectors {
 
+  private[ml] val empty: Vector = zeros(0)
```

Comment:
How about changing to Vectors.empty? https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala#L1081 https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala#L707

Reply:
Probably, we can change it later.